recipes/ega-cryptor/ega-cryptor.py

```
@@ -0,0 +1,102 @@
+    In a multiuser setting, the option --exec_dir="exec_dir"
+    can be used as the location for the peptide-shaker distribution.
+    If the exec_dir dies not exist,
+    we copy the jar file, lib, and resources to the exec_dir directory.
+    """
+    (mem_opts, prop_opts, pass_args, exec_dir) = jvm_opts(sys.argv[1:])
+    jar_dir = exec_dir if exec_dir else real_dirname(sys.argv[0])
+
+    if pass_args != [] and pass_args[0].startswith('eu'):
+        jar_arg = '-cp'
+    else:
+        jar_arg = '-jar'
```

Ruff

Use ternary operator `jar_arg = '-cp' if pass_args != [] and pass_args[0].startswith('eu') else '-jar'` instead of `if`-`else`-block

Replace `if`-`else`-block with `jar_arg = '-cp' if pass_args != [] and pass_args[0].startswith('eu') else '-jar'`

(SIM108)